src/GraphQL.Tests/Bugs/Bug2194.cs

```diff
@@ -12,7 +12,7 @@ public void Should_Print_Default_Values_Of_Arguments()
         CultureTestHelper.UseCultures(() =>
         {
             var printer = new SchemaPrinter(new Bug2194Schema(), new SchemaPrinterOptions { IncludeDeprecationReasons = false, IncludeDescriptions = false });
-            var printed = printer.Print();
+            string printed = printer.Print();
```

Comment:
I think we should leave the codebase as is with var being preferred

Reply:
Editorconfig contains rule that states var should be used for all types except primitive ones - int, string, etc. It has always been so.

Reply:
Ok